docs/rfcs/XeGPU.md

```diff
@@ -701,6 +700,332 @@ An example on how to perform transpose using load with chunk_size in SIMT flavor
 
 ```
 
+## Workgroup level XeGPU Operations
```

Comment:
It would be clearer in organization to rename `XeGPU Attributes to support work item level semantic` to `XeGPU Attributes to support work distribution` and put discussions about wg_map and sg_map as its subsections.